pages/about-us.js

```diff
@@ -81,7 +82,7 @@ export default function MeetOurTeam() {
 
       <SubSubTitle content="Email"></SubSubTitle>
       <p>
-        If you want to share project requirements, please send your message to info@condensation.io
+        If you want to share project requirements, please send your message to <Email></Email>&nbsp;
```

Comment:
I think we dont need component for that. Just use default.js and add your email variable